collections/misc/collprofiles.php

```diff
@@ -151,9 +138,8 @@ function validateForm (){
 					?>
 					
 				</form>
-				<!-- <form name="quicksearch-editor" action="javascript:void(0);" onsubmit="submitAndRedirectSearchForm('<?php echo $CLIENT_ROOT ?>/collections/editor/occurrencetabledisplay.php?displayquery=1&collid=','&q_catalognumber=', '', '', '', true, <?php echo $occid ?>); return false;"> -->
-				<form name="quicksearch" action="javascript:void(0);" onsubmit="submitAndRedirectSearchForm('<?php echo $CLIENT_ROOT ?>/collections/list.php?db=','&catnum=', '&taxa=', '&includeothercatnum=1', '&usethes=1&taxontype=2 '); return false;">
-					<button type="submit" id="search-by-catalog-number-btn" title="<?php echo (isset($LANG['IIDENTIFIER_PLACEHOLDER_LIST']) ? $LANG['IDENTIFIER_PLACEHOLDER_LIST'] : 'Occurrence ID and Record ID also accepted.'); ?>">
+				<form name="quicksearch" action="javascript:void(0);" onsubmit="submitAndRedirectSearchForm('<?php echo $CLIENT_ROOT ?>/collections/list.php?db=','&catnum=', '&taxa=', '&includecult=1&includeothercatnum=1', '&includecult=1&usethes=1&taxontype=2 '); return false;">
```

Comment:
Added &includecult=1 here.

Reply:
@themerekat This is what this branch's line 297 looks like for me:
`<input type='checkbox' name='includecult' id='includecult' value='<?php echo $CSS_VERSION_RELEASE === 'uswds' ? '1' : '0' ?>' <?php echo $CSS_VERSION_RELEASE === 'uswds' ? 'checked' : '' ?> />`

So it's only 1 if we're using the uswds styling. Probably not the best way to handle that logic, but I think it'll behave as expected at least.

However, I should probably add similar logic in collprofiles as I did in harvestparams, so that you for that idea. On it.

Reply:
Ok, so 1 = checked and 0 = not checked? That's my main question. Because I saw value=1, I thought that this was the default value, and therefore I thought unchecked = 1.

Reply:
Fixed in 4e09df7ed.